libp2p/transports/tortransport.nim

```diff
@@ -147,13 +157,13 @@ proc readServerReply(transp: StreamTransport) {.async.} =
 
 proc parseOnion3(
     address: MultiAddress
-): (byte, seq[byte], seq[byte]) {.raises: [LPError, ValueError].} =
+): (byte, seq[byte], seq[byte]) {.raises: [LPError].} =
   var addressArray = ($address).split('/')
   if addressArray.len < 2:
-    raise newException(LPError, fmt"Onion address not supported {address}")
+    raise newException(LPError, "Onion address not supported")
```

Comment:
Curious about why remove the {address} from this and other errors

Reply:
`fmt` can cause `ValueError`, and i didn't want to propagate this all the way up. also it seems unnecessary to include address here because caller knows what address it passed:

pseudo code:
```
res, err := parseOnion3(address)
if err != nil { 
   // here caller knows what address it passed and he can choose to log it, 
   // no need to included it in err
  printf("could not parse adddress %v, reason: %v", address, err)
  // stdout: "could not parse {addr}, reason: Onion address not supported"
}

```